x-pack/legacy/plugins/uptime/public/state/actions/monitor_duration.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { createAction } from 'redux-actions';
+import { QueryParams } from './types';
```

Comment:
```suggestion
import { MonitorQueryParams } from './types';
```

Borderline out of scope for this PR, but can we give this type a more descriptive name, especially since we're reusing it in more than one place and it requires a monitor ID? Or create a follow-up tech debt issue is fine with me, too.

Reply:
i have improved this.